code/style.css

```diff
@@ -1,58 +1,264 @@
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background: rgb(121, 3, 3);
 }
 
-p {
-  margin: 1px;
+h1 {
+  text-align: center;
+  font-family: 'Abril Fatface', cursive;
+  font-size: 70px;
+  color: white;
+  letter-spacing: 2px;
+  margin-top: -5px;
+  margin-bottom: -8px;
+}
+
+h2 {
+  font-weight: 700;
+  font-family: 'Abril Fatface', cursive;
+  font-size: 16px;
+  background-color: black;
+  width: 80%;
+  margin-top: -8px;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 2px;
+  color: white;
+  letter-spacing: 4px;
+  text-align: center;
+}
+
+h3 {
   font-weight: 300;
-  background-color: #fff;
+  font-family: 'Abril Fatface';
+  color: white;
+  font-size: 18px;
+  width: 80%;
+  margin: auto;
+  margin-top: 10px;
+  font-style: italic;
+  letter-spacing: 1px;
+}
+
+h4 {
+  font-weight: 0;
+  font-family: 'Abril Fatface';
+  color: rgb(225, 128, 10);
+  font-size: 28px;
+  text-align: center;
+  margin-top: auto;
+  letter-spacing: 1px;
+  font-style: italic;
+  margin-bottom: auto;
+}
+
+h5 {
+  font-weight: 0;
+  font-family: ARIAL;
+  color: black;
+  font-size: 24px;
+  text-align: center;
+  margin-top: 20px;
+  margin-bottom: 5px;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h5.reveal {
+  color: white;
+  font-size: 18px;
+}
+
+p {
+  font-weight: 0;
+  font-family: 'Abril Fatface';
+  color: white;
+  font-size: 24px;
+  margin-top: -5px;
+  letter-spacing: 1px;
+  margin-bottom: 17px;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: 10px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  height: 440px;
+  background: rgb(121, 3, 3);
   display: flex;
   justify-content: space-around;
   align-items: center;
+  margin-bottom: 20px;
+  margin-top: 20px;
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: rgb(18, 70, 28);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 27px;
   border-radius: 16px;
+  box-shadow: 3px 3px 10px rgb(41, 26, 26);
+}
+
+.killerImage {
+  border-radius: 26px;
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: rgb(18, 70, 28);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 27px;
```

Comment:
If you created another class called `deck` You could include common CSS styles from all the decks

```css
.deck {
  width: 250px;
  height: 400px;
  border-radius: 16px; 
  text-align: center; 
  padding-top: 27px; 
}
```

Then all you need is to change the styles which are specific to each deck.  You would have two classes on the deck elements in your DOM if you do this. 👍